favorites.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.IO;
+using Azure.Data.Tables;
+using Azure;
+using System.Web;
+
+namespace TableStorage
+{
+    public class Favorites
+    {
+
+        string AccountName = Environment.GetEnvironmentVariable("AccountName");
+        string TableFavorite = Environment.GetEnvironmentVariable("TableFavorites");
```

Comment:
Something that we often do for our database configurations is create a separate class from our database names using something called an IOptions pattern. 

I would recommend incorporating this so this so that you don't have to do `Environment.GetEnvironmentVariable` for every database name. (: 

https://code-maze.com/aspnet-configuration-options/
https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.options.ioptions-1?view=dotnet-plat-ext-6.0

Reply:
I'll need to figure this one out

Reply:
figured out :)